src/FSharp.Configuration.Runtime/FSharp.Configuration.Runtime.fsproj

```diff
@@ -4,7 +4,7 @@
   <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
   <PropertyGroup>
     <OutputType>Library</OutputType>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
Why you  decided to drop full framework support?

Reply:
Yea, that works with `netstandard2.0`, too. I was playing around with it.